### ISSUE TITLE
Some Lua errors when using the script

### ISSUE THREAD

**Issue by NanXiao** [jzwinck/redis-wireshark#1]
Hi, John:

```
I have used your script, but it seems not work well. There are some erros I have encountered:

Lua Error: d:\Program Files\Wireshark\redis.lua:52: attempt to index local 'submatch' (a nil value)
Lua Error: d:\Program Files\Wireshark\redis.lua:33: attempt to index local 'line' (a nil value).

I am not very familiar with Lua, so I can't find out the root cause. Could you help to check it? Thanks very much in advance!
```

Best Regards
Nan Xiao

**Comment by georgasa** [comment]
Hello,

with Wireshhark Version 2.2.7 (v2.2.7-0-g1861a96) we get Lua: syntax error during precompilation of redis-wireshark.lua with unspupported '<' symbol at the very start of your script:

**<**!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
  <link rel="dns-prefetch" href="https://assets-cdn.github.com">
  <link rel="dns-prefetch" href="https://avatars0.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars1.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars2.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars3.githubusercontent.com">
  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">
...

Any ideas how to overcome it?

Thanks!

**Comment by georgasa** [comment]
Hi @jzwinck

I just found that you have provided already a working solution. Thanks!

https://devhub.io/repos/jzwinck-redis-wireshark

**Comment by johnzeng** [comment]
@jzwinck You can also close this issue. It's fixed by my PR #6

**Comment by jzwinck** [comment]
Thanks @johnzeng for #6.
